qcodes/instrument_drivers/Minicircuits/USBHIDMixin.py

```diff
@@ -19,32 +19,29 @@
 class USBHIDMixin(Instrument):
     """
     Args:
-        instance_id (str): The id of the instrument we want to connect. If
-            there is only one instrument then this is an optional argument.
-            If we have more then one instrument, quarry their ID's by calling
-            the class method 'enumerate_devices'
-        timeout (float): Specify a timeout for this instrument
+        instance_id: The id of the instrument we want to connect to. If
+            there is only one instrument, then this argument is optional.
+            If more than one instrument happen to be connected, use
+            `enumerate_devices` method to query their IDs
+        timeout: Specify a timeout for this instrument in seconds
     """
-    # The following class attributes are set by subclasses
+    # The following class attributes should be set by subclasses
     vendor_id = 0x0000
     product_id = 0x0000
 
     @staticmethod
     def _check_hid_import():
-
         if os.name != 'nt':
-            raise ImportError("""This driver only works in Windows.""")
+            raise ImportError("This driver only works on Windows.")
 
         if hid is None:
-
             raise ImportError(
                 "pywinusb is not installed. Please install it by typing "
                 "'pip install pywinusb' in a qcodes environment terminal"
             )
 
-    def __init__(self, name, instance_id: str=None, timeout: float=2,
-                 **kwargs) ->None:
-
+    def __init__(self, name: str, instance_id: str=None, timeout: float=2,
+                 **kwargs):
```

Comment:
wow you are so up to date. This was only in September:
https://github.com/python/mypy/pull/5677